src/maps/cc-map.js

```diff
@@ -378,10 +380,9 @@ export class CcMap extends withResizeObserver(LitElement) {
         }
 
         :host(:not(:empty)) .legend {
-          background-color: #f1f5ff;
+          background-color: var(--color-bg-neutral);
           box-shadow: inset 0 6px 6px -6px #a4b1c9;
           box-sizing: border-box;
-          color: #2e2e2e;
```

Comment:
I could replace `#2e2e2e` color with `var(--color-text-default)` but I did not really see the point since it will be the default color for all texts soon.